.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "./node_modules/gts/",
+  "env": {
+    "mocha": true
+  },
+  "rules": {
+    "node/no-missing-require": ["off"], 
+    "node/no-unpublished-require": ["off"], 
+    "no-unused-vars": ["off"], 
```

Comment:
Ditto - why is this disabled?

Reply:
Ah NVM, saw your comment above - but can you file a GitHub issue for tracking?

Reply:
See PR description